bindings/python/quokka/utils.py

```diff
@@ -177,3 +183,36 @@ def parse_version(version: str) -> tuple[int, int, int]:
         )
 
     return parsed
+
+
+def find_reg_access(
+    reg_id: int, access_mode: RegAccessMode, instructions: Iterable[Instruction]
```

Comment:
Can't we also provide the regiser by its name ? (and resolving its id within the function) ? So the type would be ``int | str`` ?